.github/workflows/pre-commit.yml

```diff
@@ -0,0 +1,14 @@
+name: pre-commit
+
+on:
+  pull_request:
+  push:
+    branches: ['master']
+
+jobs:
+  pre-commit:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: Alfresco/alfresco-build-tools/.github/actions/setup-checkov@master
```

Comment:
better reference a specific tag of alfresco-build-tools for reproducible builds rather than the master branch